docs/guides/primitive-input-output.ipynb

```diff
@@ -356,6 +356,19 @@
     "print(f'The expectation values measured from this PUB are: \\n{result[0].data.evs}')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### How the Estimator calculates the error bar\n",
+    "\n",
+    "Estimator queries return not only an estimate of the mean of the observables passed in the input PUBs (the `evs` field of the `DataBin`), but they also attempt to deliver information on the \"error bar\" associated with those estimates. All estimator queries will populate the `stds` field with a quantity like the standard error of the mean for each expectation value, but some error mitigation options produce additional information.\n",
+    "\n",
+    "Consider a single observable $O$. In the absence of ZNE, you can think of each shot of the Estimator execution as providing a point estimate of the expectation value $\\langle O \\rangle$. If the pointwise estimates are in a vector `Os`, then the value returned in `ensemble_standard_error` is equivalent to `np.std(Os) / np.sqrt(shots)`, which treats all shots as part of a single ensemble. If you requested gate twirling (`twirling.enable_gates = True`), you can sort the pointwise estimates of $\\langle O \\rangle$ into groups that share a common twirl. Call these grouped estimates `O_twirls`, and there are `num_randomizations` (number of twirls) of them. Then `stds` is the standard error of the mean of `O_twirls`, i.e. `np.std(O_twirls) / np.sqrt(num_randomizations)`. When you do not enable twirling, `stds` and `ensemble_standard_error` are equal.\n",
```

Comment:
@kaelynj : Blake noted on his Issue that maybe it would be better to use mathematical formulae instead of `np.std(Os) / np.sqrt(shots)` and `np.std(O_twirls) / np.sqrt(num_randomizations)` - thoughts?

Reply:
I agree, it'll make reading the math much easier if they're rendered in LaTeX format

Reply:
I know how to do the square root but not sure how to represent np.std in LateX?

Reply:
Also unsure if num_randomizations should be represented as num (superscript: randomizations) -- same with 0_twirls